IDEAS/Buildings/Components/Interfaces/PartialSurface.mo

```diff
@@ -32,6 +32,9 @@ partial model PartialSurface "Partial model for building envelope component"
     "Static (steady state) or transient (dynamic) thermal conduction model"
     annotation(Evaluate=true, Dialog(tab = "Dynamics", group="Equations"));
 
+  parameter Boolean errorLowPerformance = true
```

Comment:
I don't think that we need this parameter here? You can directly set multiLayer.errorLowPerformance = glazing.errorLowPerformance without needing this parameter, which the user would then have to worry about.

Reply:
Ok, great, changed!